client.js

```diff
@@ -186,62 +201,63 @@ class Client {
    * Request events based on the provided query parameters.
    *
    * The `query.flags` field uses bitmask values similar to an enum:
-   *   0 = None
-   *   1 = NewestFirst
-   *   2 = TimeRangeBeginExclusive
-   *   4 = TimeRangeEndExclusive
-   *   8 = UseLogStampForTimeRange
-   *
-   * The `query.senderConditions` field can be used to filter by event sender (Source).
-   * The `query.dataConditions` field can be used to filter by data fields (key-value patterns).
    *
-   * For additional information:
-   *   https://cdpstudio.com/manual/cdp/cdplogger/eventlogreader.html#Flags-enum
+   *   0 = None  
+   *   1 = NewestFirst  
+   *   2 = TimeRangeBeginExclusive  
+   *   4 = TimeRangeEndExclusive  
+   *   8 = UseLogStampForTimeRange  
+   * 
+   *      For additional information:
+   *   https://cdpstudio.com/manual/cdp/cdp2sql/logmanager-eventquery.html#Flags-enum
    *   https://cdpstudio.com/manual/cdp/cdplogger/eventlogreader.html#cdp-event-code-flags
    *
-   * We also support mapping the `evt.code` field to a human-readable string
-   * with `getEventCodeDescription()`.
+   * In addition, the user can simply supply the following properties in the query object:
+   *
+   *   - **senderConditions**: An array of sender strings (exact matches by default).
+   *   - **dataConditions**: An object where each key is a data field name and the value can be:
+   *       - A string (defaults to an exact match),
+   *       - An array of strings,
+   *       - An object (or array of objects) with properties:
+   *           - `value`: the string value to match,
+   *           - `matchType`: either `"exact"` (default) or `"wildcard"`.
```

Comment:
I don't see any place where the matchType can be customized. Is the doc here wrong?